contracts/PLCR/PLCRVoting.sol

```diff
@@ -1,16 +1,19 @@
-pragma solidity ^0.5.0;
+pragma solidity >=0.5.0 <0.6.0;
+
+import "../../node_modules/zos-lib/contracts/Initializable.sol";
```

Comment:
before we commit to main chain we'll want to copy in the Initializable contract into the project and reference it there rather than via node_modules

Reply:
Would we have to change the `zos` commands then? Or would we have to programmatically control the upgrade process since the `zos` command always references the Initializable contract in the `node_modules`?